### ISSUE TITLE
Get challenge service not available

### ISSUE THREAD

**Issue by mentioneddd** [decentralized-identity/ion#328]
The get challenge service seems to be unavailable.

 try {
      didUri = await did.getURI('short');
      anchorRequestBody = await did.generateRequest();
      anchorRequest = new ION.AnchorRequest(anchorRequestBody);
      anchorResponse = await anchorRequest.submit();
    } catch (e) {
      console.error("Error anchoring DID:", e);
      throw new Error(`Error at DID anchoring stage: Error: ${e.message}, Request Body: ${anchorRequestBody}, Request: ${anchorRequest}, Response: ${anchorResponse}`);
    }

the anchorResponse is undefinied and throws the error: Get challenge service not available

____
Generated DID: did:ion:EiBIn2lbWCvXj5Ci445YhJp6BzwF09ugYEaWOXU5xTxlYg
Getting challenge from: https://beta.ion.msidentity.com/api/v1.0/proof-of-work-challenge

Website shows: {"requestId":"9def974b929c460966b9b2f54a06ac0b","date":"Thu, 31 Aug 2023 21:37:32 GMT","mscv":"idcsgbfC2D0iCTr8.0","error":{"code":"not_found","message":"resource was not found"}}

**Comment by csuwildcat** [comment]
Looks like their create node endpoint is down, which we can help communicate to them. We're spinning ours up at Block now for folks to have another option.

**Comment by bogdanshlb** [comment]
> Looks like their create node endpoint is down, which we can help communicate to them. We're spinning ours up at Block now for folks to have another option.

How come that this is not monitored by them? The service has been unavailable for at least 48 hours.

**Comment by csuwildcat** [comment]
Adding @dangodb, who works at Microsoft

**Comment by mentioneddd** [comment]
> > Looks like their create node endpoint is down, which we can help communicate to them. We're spinning ours up at Block now for folks to have another option.
> 
> How come that this is not monitored by them? The service has been unavailable for at least 48 hours.

Agreed, questioned my general usage of ION to identify my users. However it's a great cause and there is no real alternative to identfy users with bitcoin rn. I hope they'll fix it and improve the project further. They def. will need to drive the cost down eventough it's cheap compared to save data directly onchain it def should be cheaper. The bitcoin wizard community opened quite some oppertunity to build onchain. Maybe even putting the dids in open lightning channels or smth.

**Comment by bogdanshlb** [comment]
Any news?

**Comment by mentioneddd** [comment]
@csuwildcat How can we use blocks service instead of theirs?

**Comment by csuwildcat** [comment]
Yeah, we can make it available, we just need a similar gating mechanism that they have at MSFT for protecting against spam

**Comment by mentioneddd** [comment]
@csuwildcat Yes please, incredible for me that they've simply shut it down.  -- Please let everyone here know once you guys have something usable for us. Much love, thanks!

**Comment by mentioneddd** [comment]
@csuwildcat Can we get an update on the time this will take? This is really important. I'm sure this drives many people away from actually using ION and creating real usecases.

**Comment by csuwildcat** [comment]
We won't have our ION node exposed for another month, due to timelines for prod exposure. However, we are working on a simplified DID Method that could be used sooner. I can provide further details in a couple weeks.

**Comment by mentioneddd** [comment]
> We won't have our ION node exposed for another month, due to timelines for prod exposure. However, we are working on a simplified DID Method that could be used sooner. I can provide further details in a couple weeks.

@csuwildcat  please drop any update, if available.

**Comment by sakalis** [comment]
@csuwildcat  any update on this please?

**Comment by csuwildcat** [comment]
@sakalis Microsoft is no longer doing a free anchoring service, and we'll be soon offering free did:dht support, so we encourage folks to shift focus to that: https://did-dht.com/

**Comment by sakalis** [comment]
@csuwildcat so we should wait new version of ion-tools ?

**Comment by Tahlil** [comment]
@csuwildcat does this mean the Ion project has been sunsetted?
